### DISCUSSION TITLE
Importing fonts to 3rd party vendors

### DISCUSSION THREAD

**Discussion by ttyler-twilio** [twilio-labs/paste#2310] [Q&A] [answered]
Hey folks,

I am wanting to import the Whitney fonts to a 3rd party vendor to use in the page styling. Do we have a CDN link or anything available that I can directly import these fonts from to be used? I see this discussion here providing direct file links to the `.otf` files, however I need something similar to the following if possible as this 3rd party vendor does not allow for direct file uploads.

`@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed');`

**Comment by nkrantz** [comment, marked as answer]
Hi @ttyler-twilio - could you provide some more context on why you need access to the Whitney fonts? For some background info, we use **Inter** as our product design language typeface, not Whitney. Whitney is only licensed to be used on certain domains, and the company is actually moving away from Whitney altogether this year.

**Reply by shleewhite** [reply]
Hi @ttyler-twilio, just checking in on this again. Could you provide more context?

**Reply by ttyler-twilio** [reply]
Hey folks, sorry I didn't see this come through. The designs to re-skin our status page specifically use Whitney in the Figma designs. I was unaware that Whitney is being fazed out in favor of Inter. In that case, is there a CDN link I can use to pull inter from somewhere? I managed to pull Whitney from a blog post I found, but given that it's being phased out I will need a less hacky way to pull in Inter if that is possible.

**Reply by SiTaggart** [reply]
Clarifying question? Are you using Paste?

If so, you will automatically get the fonts downloaded for you when you use the Theme provider. That also future proofs you for when we switch the new Twilio font in the future.

**Reply by ttyler-twilio** [reply]
We are not currently using react in this iteration. We are unsure if we can get react/paste into this vendor tool but will be trying so next sprint to do so.

**Reply by shleewhite** [reply]
@ttyler-twilio In that case, [the manual installation page on our docs site](https://paste.twilio.design/introduction/for-engineers/manual-installation#how-to-load-the-right-font) has information about loading the right fonts.

To use the fonts, you need to add these to the `<head />`:
```html
<link rel="preconnect" href="https://assets.twilio.com" crossorigin />
<link rel="stylesheet" href="https://assets.twilio.com/public_assets/paste-fonts/main-1.2.0/fonts.css">
```

**Note:** we are changing the fonts soon, so you will have to update the version manually.

**Reply by ttyler-twilio** [reply]
Awesome, thanks a lot for your help! @shleewhite
